README.md

```diff
@@ -69,8 +69,12 @@ We could change third-party context’s StorageKey to be the first-party one so
 
 Service workers have [cache-based history sniffing attacks](https://www.ndss-symposium.org/wp-content/uploads/ndss2021_1C-2_23104_paper.pdf). Extending cross-site unpartitioned storage access to service workers would open up increased vulnerabilities and be somewhat confusing due to the way FetchEvent and other background events are not tied to an endpoint, thus first-party Service Workers will not be exposed in third-party contexts after an rSA call.
 
+### Shared/Dedicated Workers
+
+Shared and Dedicated Workers have access to SameSite=Strict cookies. This API does not otherwise grant access to those cookies in a third-party context, so it should not allow access to first-party worker pools.
```

Comment:
Can you move this note into the P&S Considerations and adapt it to note that while we want to support Shared Workers we should ensure that they don't grant access to SameSite=Strict in cross-site contexts?

Reply:
You could leave the mention of Dedicated Workers here and point to https://github.com/privacycg/storage-access/issues/102 as a way to request cross-site cookie access in DWs.